src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -283,6 +283,27 @@ void SimulateVRF(EnergyPlusData &state,
         if (state.dataHVACVarRefFlow->VRF(VRFCondenser).CondenserType == DataHeatBalance::RefrigCondenserType::Water)
             UpdateVRFCondenser(state, VRFCondenser);
     }
+    // yujie: update coil and IU evaporating temperature, also keep coil RTF updated with the condenser side cycling ratio, for the FluidTCtrl model
```

Comment:
Current practice is to leave names out of comments.

Reply:
I'll remove the name